css/style.css

```diff
@@ -0,0 +1,3154 @@
+    height: 48.91px;
+  }
+
+  .speedo__image-ellipse {
+    width: 30.17px;
+    height: 30.01px;
+    left: 17px;
+    top: 12px;
+  }
+
+  .speedo__image-arrow {
+    width: 30px;
+    height: 16px;
+    left: 28px;
+    top: 20px;
+  }
+
+  .speedo__title {
+    margin-top: -53px;
+    padding-left: 70px;
+  }
+
+  .speedo__title-numb {
+    font-size: 50px;
+  }
+
+  .speedo__title-text {
+    font-size: 20px;
+    padding-bottom: 15px;
+  }
+
+  .speedo__desc {
+    font-size: 12px;
+    padding-left: 70px;
+  }
+
+  .description-characteristic__road {
+    padding-right: 18px;
+  }
+
+  .road__image-stroke {
+    width: 41.56px;
+    height: 64.45px;
+    padding-top: 0px;
+  }
+
+  .road__image-strokeDotted {
+    width: 37.83px;
+    height: 60.72px;
+    top: 6px;
+  }
+
+  .road__image-ellipseRightTop {
+    width: 10.03px;
+    height: 10.03px;
+    top: 5px;
+    left: 35px;
+  }
+
+  .road__image-ellipseRightTopIn {
+    display: none;
+  }
+
+  .road__image-ellipseLeftBottom {
+    top: 62px;
+    width: 10.03px;
+    height: 10.03px;
+  }
+
+  .road__title-numb {
+    padding-left: 19px;
+    font-size: 50px;
+    margin-top: -3px;
+  }
+
+  .road__title-text {
+    font-size: 20px;
+    padding-bottom: 15px;
+  }
+
+  .road__desc {
+    padding-left: 60px;
+    font-size: 12px;
+  }
+
+  .description-characteristic__testDrive {
+    display: none;
+  }
+
+  .description-characteristic__button {
+    display: none;
+  }
+
+  .description-characteristic__wrap-phone {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 57px;
+  }
+
+  .description-characteristic__testDrive-phone {
+    display: flex;
+    align-items: center;
+    padding-right: 80px;
+  }
+  
+  .testDrive-phone__image {
+    position: relative;
+    padding-right: 40px;
+  }
+  
+  .testDrive-phone__image-ellipse {
+    position: absolute;
+    width: 22px;
+    height: 22px;
+    top: -11px;
+    left: 10px;
+  }
+  
+  .testDrive-phone__image-tesla {
+    position: absolute;
+    width: 14.18px;
+    height: 8.23px;
+    left: 14px;
+    top: -1px;
+    animation-duration: 3s;
+    animation-name: rotatePhone;
+    animation-iteration-count: infinite;
+    animation-direction: alternate;
+    transform-origin: 50% 0%;
+  }
+  
+  @keyframes rotatePhone {
+    from {
+      transform: rotate(0deg);
+    }
+  
+    to {
+      transform: rotate(-230deg);
+    }
+  }
+  
+  .testDrive-phone__image-dott {
+    position: absolute;
+    width: 2.58px;
+    height: 2.58px;
+    left: 20px;
+    top: -3px;
+  }
+  
+  .testDrive__title-phone {
+    margin: 0px;
+    padding-left: 0px;
+    font-size: 16px;
+    font-family: "MontserratBold";
+    color: #E51937;
+  }
+  
+  .description-characteristic__button-phone {
+    margin: 0px;
+    color: white;
+    background-color: #E51937;
+    display: flex;
+    align-self: center;
+    font-family: "MontserratBold";
+    font-size: 16px;
+    border: 2px solid #E51937;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#E51937" to be "#e51937" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/tesla_project/pullRequest?prid=5520131)